dist/stat_data.py

```diff
@@ -365,6 +365,7 @@ def __init__(self, name, desc, flags=''):
     CursorStat('cursor_next_skip_lt_100', 'cursor next calls that skip less than 100 entries'),
     CursorStat('cursor_next_skip_total', 'Total number of entries skipped by cursor next calls'),
     CursorStat('cursor_prev', 'cursor prev calls'),
+    CursorStat('cursor_prev_hs_tombstone', 'cursor prev calls that were the result of a history store tombstone'),
```

Comment:
Can you make this: "cursor prev calls that skip due to a globally visible history store tombstone"?